src/opening-period/page/EditOpeningPeriodPage.test.tsx

```diff
@@ -687,4 +658,76 @@ describe(`<EditNewOpeningPeriodPage />`, () => {
       expect(mockHistoryPush).toHaveBeenCalledWith(testResourcePageUrl);
     });
   });
+
+  it('should disable rule start when "Parillinen" frequency is selected', async () => {
```

Comment:
It seems we have multiple places where we could have this test, and I don't really have a strong opinion which place would be best:

1. EditNewOpeningPeriodPage
2. CreateNewOpeningPeriodPage
3. OpeningPeriodForm
4. Rule.test.tsx

What do you think which place should have which tests?

Reply:
I could move rule test under OpeningPeriodForm or Rule.test.tsx? 🤔